docs/release-notes/8.6.asciidoc

```diff
@@ -0,0 +1,99 @@
+* Enables you to bulk-add up to 4000 events to Timeline ({pull}142737[#142737]).
+* Enables alert suppression per rule execution for custom query rules ({pull}142686[#142686]).
+* Improves role-based access controls for {kib} users performing response actions ({pull}142825[#142825]).
+
+[discrete]
+[[bug-fixes-8.6.0]]
+==== Bug fixes and enhancements
+* Adds the *View indicators* button to the Threat Intelligence card ({pull}145125[#145125]).
+* Updates the guided onboarding walkthrough ({pull}144016[#144016], {pull}143598[#143598]).
+* Improves the interface for creating rule exceptions and shared exception lists ({pull}144575[#144575]).
+* Adds cases metadata in the Cases panel on the alert details page ({pull}144430[#144430]).
+* Improves the UX for managing {ml} jobs while managing {ml} rules ({pull}144080[#144080]).
+* Enables you to run {ml} jobs from the Notable Anomalies table ({pull}142861[#142861]).
+* Updates the take action UI for charts on the Hosts, Users, and Network pages ({pull}138369[#138369]).
+* Adds a *Respond* button to the Host Details page for hosts with an {agent} installed ({pull}143988[#143988]).
+* Allows you to add up to three new terms to New Terms rule queries, enabling you to create alerts when multiple new terms appear in the same event ({pull}143943[#143943]).
+* Allows you to launch Timeline from the Entity Analytics dashboard by clicking alert counts ({pull}143841[#143841]).
+* Adds missing TLP Marking badges to the Indicators table and Indicator details flyout ({pull}143431[#143431]).
+* Ensures the empty state of the Indicators page does not appear when threat intelligence integrations are installed ({pull}143328[#143328]).
+* Turns the anomalies count on the Entity Analytics dashboard into a link that jumps down the page to the Anomalies table ({pull}143085[#143085])
+* Pre-selects the `threat` category when you open the Fields browser ({pull}142698[#142698]).
+* Adds a `copy to clipboard` action for indicators in the Indicators table ({pull}142675[#142675]).
+* Adds a `User risk classification` column to the Users table ({pull}142610[#142610]).
+* Adds a label to the Indicators page that states when it was last updated ({pull}142560[#142560]).
+* Specifies that links from the Threat Intelligence page to the Integrations page should open the Threat Intelligence integrations category ({pull}142538[#142538]).
+* Enables full-screen mode on the Indicators table ({pull}142519[#142519]).
+* Implements the standard search bar and date picker on the Threat Intelligence page ({pull}142336[#142336]).
+* Updates the design of the Rule Exceptions page ({pull}142289[#142289]).
+* Displays comments for expanded items in the Action history page ({pull}141938[#141938]).
+* Adds HTTP 409 conflict response status codes to error messages for several API requests ({pull}146389[#146389]).
+* Adds the new Data Exfiltration Detection (DED) integration package (https://github.com/elastic/integrations/pull/4486[#4486]).
```

Comment:
Adding a release note for this small change (no change to the existing line above it, it just happened to be where I wanted to add the new note):
```suggestion
* Adds the new Data Exfiltration Detection (DED) integration package (https://github.com/elastic/integrations/pull/4486[#4486]).
* Renames the sorting toggle on the Rules page from *Technical preview* to *Advanced sorting* (https://github.com/elastic/kibana/pull/144733[#144733]).
```